adal/build.gradle

```diff
@@ -81,7 +81,20 @@ android {
     }
 
     flavorDimensions "main"
-
+    afterEvaluate {
+        // Gradle 7.5 and Android Gradle Plugin 7.4.2 doesn't rename the file (see 'outputFileName' above)
+        // Adding this work around to have the file properly renamed.
+        tasks.named("assembleDistRelease").configure {
+            def buildFile = file("$buildDir/outputs/aar/${archivesBaseName}-dist-release.aar")
+            print "Build file $buildFile"
+            doLast {
+                println "Renaming build file $buildFile to '$buildDir${File.separator}outputs${File.separator}aar${File.separator}${archivesBaseName}-${version}.aar'"
+                if (!buildFile.renameTo("$buildDir${File.separator}outputs${File.separator}aar${File.separator}${archivesBaseName}-${version}.aar")) {
+                    println "Rename failed!"
+                }
+            }
+        }
+    }
```

Comment:
Check if we can do this in the maven release pipeline itselft, instead of the build.gradle